content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
    :guilabel:`Virtual Locations/Scrap` location.
 
-Scrap manufacturing components
-==============================
-
-To scrap components during the manufacturing process, begin by navigating to
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+.. _scrap_components/scrap-window:
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Open the Scrap pop-up window
+============================
 
-.. image:: scrap_manufacturing/scrap-button.png
-   :align: center
-   :alt: The scrap button on a manufacturing order.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window. Once
+opened, the pop-up window can be filled out in the same way, regardless of how it was accessed.
+However, the method for accessing the window differs depending on the type of scrapping taking
+place.
 
-From the :guilabel:`Product` drop-down menu on the :guilabel:`Scrap` pop-up window, select the
-component that is being scrapped, then enter the quantity in the :guilabel:`Quantity` field.
-Finally, click :guilabel:`Done` to scrap the component.
+Scrap component from Manufacturing
+----------------------------------
 
-.. image:: scrap_manufacturing/scrap-pop-up.png
-   :align: center
-   :alt: The Scrap pop-up window.
+To scrap a component from an |MO|, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, and then select an |MO|. At the top of the |MO|, click the
+:guilabel:`Scrap` button to open the :guilabel:`Scrap` pop-up window.
```

Comment:
```suggestion
Operations --> Manufacturing Orders`, and then select an |MO|.

At the top of the |MO|, click the :guilabel:`Scrap` button to open the :guilabel:`Scrap` pop-up
window.
```

Totally optional, but since this section is a little short, maybe you can include an image of where the scrap button is. I heard from some ppl in support that they like seeing more pictures, as well! If you decide to do this, I recommend adding screenshots to all the subsections (:

![image](https://github.com/odoo/documentation/assets/23611711/65d02070-aa4b-4be4-b018-8e3b800db4a8)

Reply:
I try not to include pictures that only point out a single button, especially when it's very easy to find.